pkgs/development/compilers/purescript/purescript/default.nix

```diff
@@ -0,0 +1,53 @@
+{ pkgs ? import <nixpkgs> {} }:
+
+# from justinwoo/easy-purescript-nix
```

Comment:
If you're planning on maintaining this packaging in nixpkgs (which we would like if you're  contributing it :)) this comment doesn't need to be here.

```suggestion
```

Reply:
I plan to maintain my own repo but upstream these changes regularly, so should I just keep this here?

Reply:
Sure, it won't hurt.